Code/GraphMol/MolStandardize/catch_tests.cpp

```diff
@@ -601,3 +601,15 @@ TEST_CASE("github #2965: molecules properties not retained after cleanup", "[cle
 	CHECK(x.getVal<std::string>("testing_prop") == "1234");
   }
 } 
+
+TEST_CASE("github #2970: chargeParent() segmentation fault when standardization is skipped i.e. skip_standardize is set to true") {
+    std::unique_ptr<RWMol> m(SmilesToMol("COC=1C=CC(NC=2N=CN=C3NC=NC23)=CC1"));
```

Comment:
Just so you're aware, in newer code we're using this way to construct molecules in tests:
```suggestion
    auto m = "COC=1C=CC(NC=2N=CN=C3NC=NC23)=CC1"_smiles;
```
This takes advantage of the C++11 feature to make the code shorter and, IMO, more readable.
The result is also a `std::unique_ptr<RWMol>`, so you don't need to worry about memory leaks.
If you're curious about how this is implemented: https://github.com/rdkit/rdkit/blob/master/Code/GraphMol/SmilesParse/SmilesParse.h#L113